doc/Changelog.md

```diff
@@ -3,6 +3,14 @@ Changelog {#Changelog}
 
 # git master
 
+* [45](https://github.com/BlueBrain/Brion/pull/45):
+  Made targets optional. brain::Circuit::getGIDs() does not require targets
```

Comment:
Too long :) Usually the brief of the commit, if you want to write more, put in the commit message in a new paragraph.